docs/cow-protocol/concepts/how-it-works/how-intents-are-formed.mdx

```
@@ -0,0 +1,318 @@
+An _intent_ in CoW Protocol is a signed message that represents a user's wish to trade. It doesn't execute a trade directly — instead, it delegates execution to [solvers](../introduction/solvers) who find the optimal path.
+
+:::caution
+
+This document explains the anatomy of intents and interaction with the API at a low level.  
+For practical use of the protocol, it is recommended to use high-level tools such as the **[CoW SDK](../../reference/sdks/cow-sdk)**.
+
+:::
+
+Forming an intent follows three stages:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)